README.md

```diff
@@ -268,6 +265,16 @@ From @garthk, on a 2GHz core you can roughly expect:
     rounds=31: 2-3 days/hash
 
 
+## A Note on Timing Attacks
+
+Because it's come up multiple times in this project, and other bcrypt projects, it needs to be said- bcrypt comparator functions are not susceptible to timing attacks. They are not susceptible because the comparison function is comparing one hash to another. Basically it boils down to you know nothing about the password you're trying to attack by trying to use a timing attack.
```

Comment:
> Basically it boils down to you know nothing about the password you're trying to attack by trying to use a timing attack.

This sentence reads strangely. Not sure what my exact recommendation is other than maybe stating in a sentence or two the _gist_ of the reasoning in bcrypt-ruby#43.